osbs/api.py

```diff
@@ -691,6 +694,13 @@ def _do_create_prod_build(self,
                               target=None,
                               **kwargs):
 
+        if git_uri == _REQUIRED_PARAM:
+            raise OsbsException('required parameter git_uri missing')
+        if git_ref == _REQUIRED_PARAM:
+            raise OsbsException('required parameter git_ref missing')
+        if git_branch == _REQUIRED_PARAM:
+            raise OsbsException('required parameter git_branch missing')
+
```

Comment:
would be better to first check all params which are missing and then raise,
because if all 3 are missing users will always get just the 1st missing

Reply:
Fixed.